### DISCUSSION TITLE
红米AX6，使用天灵源码仓库编译后，接网线，用WIFI时速度异常

### DISCUSSION THREAD

**Discussion by chika0801** [immortalwrt/immortalwrt#720] [Q&A]
源码仓库是 https://github.com/1715173329/imoutowrt/tree/master-ax3000-5.15

路由器是AX6，编译时保持默认插件配置（不添加如ssrp），在路由器上刷好（分别在2个AX6上测试过）。

故障现象：本地网络1000M下行，上行50M。电脑接网线到路由器，跑speedtest国内节点，下行60M，上行3M。

手机用WIFI到路由器，下行恢复成900M，上行0.3M。

我在4月份时编译了固件，没有此问题。最近更新xray core后编译，发现了这现象。

**Comment by 1715173329** [comment]
This is not officially support by ImmortalWrt.
